lib/screens/settings/settings_screen.dart

```diff
@@ -56,6 +52,33 @@ class SettingsScreen extends StatelessWidget {
     );
   }
 
+  void _popdialog(BuildContext theContext){
```

Comment:
I'd change the name to be more descriptive, e.g `_showConfirmationDialog`.

Reply:
Sure @bartekpacia will change it soon.